pkg/receiver/telegrafreceiver/converter.go

```diff
@@ -157,6 +189,46 @@ func setName(pm *pdata.Metric, name string, key string, separateField bool) {
 	}
 }
 
+func newDoubleCounter(key string, value float64, t time.Time, separateField bool) pdata.Metric {
+	pm := pdata.NewMetric()
+	pm.SetDataType(pdata.MetricDataTypeDoubleSum)
+	// "[...] OTLP Sum is either translated into a Timeseries Counter, when
+	// the sum is monotonic, or a Gauge when the sum is not monotonic."
+	// https://github.com/open-telemetry/opentelemetry-specification/blob/7fc28733eb3791ebcc98fed0d858a7961f1e95b2/specification/metrics/datamodel.md#opentelemetry-protocol-data-model
```

Comment:
Is the comment correct?

Reply:
It's not very accurate but it contains the quote from above.